OpenRA.Game/OpenRA.Game.csproj

```diff
@@ -73,6 +73,9 @@
     <StartAction>Project</StartAction>
     <StartArguments>Game.Mod=ts</StartArguments>
   </PropertyGroup>
+  <PropertyGroup>
+    <ApplicationManifest>app.manifest</ApplicationManifest>
```

Comment:
The issue is that make.cmd and the csproj aren't used by the packaging scripts, which run on linux using the makefile and `csc`.